OpenRA.Mods.Common/Warheads/CreateEffectWarhead.cs

```diff
@@ -43,77 +43,14 @@ public class CreateEffectWarhead : Warhead
 		[Desc("Chance of impact sound to play.")]
```

Comment:
Can remove the following usings from this file:
using System.Linq;
using OpenRA.Mods.Common.Traits;
using OpenRA.Primitives;